live-examples/js-examples/string/meta.json

```diff
@@ -1,5 +1,19 @@
 {
     "pages": {
+        "stringIterator": {
```

Comment:
It looks like this PR has the same issues with meta.json as https://github.com/mdn/interactive-examples/pull/1070.